package.json

```diff
@@ -25,6 +25,8 @@
     "test": "npm run build && npm run tests-only",
     "storybook": "start-storybook -p 6006",
     "storybook:css": "npm run build:css && start-storybook -p 6006 -c .storybook-css",
+    "happo": "npm i --no-save happo-plugin-puppeteer && happo",
+    "happo-ci": "happo-ci-travis",
```

Comment:
why is this a separate binary? what does it do differently?

Reply:
The first command will create a single report (set of screenshots) and can be run locally as well as in CI. The second command is designed to run in CI. It will (if needed) check out the commit that the PR is based on (usually a commit on master), run the first command, then check out the tip of the PR, run the first command again. As a final step, it will compare the two reports which will create a diff between the two commits.